omnibus/package-scripts/agent-rpm/posttrans

```diff
@@ -108,9 +108,48 @@ install_method:
   installer_version: $installer_version
 "
     echo "$install_info_content" > $CONFIG_DIR/install_info
-    chown -R dd-agent:dd-agent ${CONFIG_DIR}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why was this `chown` removed?